.github/ISSUE_TEMPLATE/new-release-checklist.md

```diff
@@ -0,0 +1,30 @@
+---
+name: New Release Checklist
+about: Maintainers checklist for publishing new releases
+title: New Release Checklist - vX.X.0
+labels: ''
+assignees: ''
```

Comment:
Sure. Though, ideally, we can have anyone on the team do this, as long as they have all the info on the checklist. Since this is an Issue template, not a PR one, it doesn't need approval or anything. Though, sign off might be a good idea.

Reply:
Agree that anyone should do it, but for the first few, I'd want to be involved. The fewer gatekeepers we have, the better. Signoffs => lost velocity.

Reply:
```suggestion
assignees: 'fundthmcalculus tmarkovski'
```